salt/cloud/clouds/proxmox.py

```diff
@@ -602,7 +602,8 @@ def create(vm_):
     elif 'private_ips' in data:
         ip_address = six.text_type(data['private_ips'][0])  # first IP
     else:
-        raise SaltCloudExecutionFailure  # err.. not a good idea i reckon
+        # err.. not a good idea i reckon
```

Comment:
I know this isn't your code @BrianSidebotham, but can you remove this comment? It's not really serving much purpose here.

Reply:
@rallytime Sure no problem. I'll update on Monday when I'm back in the office. Thanks.